encrypted-media-respec.html

```diff
@@ -3969,33 +3969,29 @@
         </section>
         <section>
           <h4>Mitigations</h4>
-          <p>In summary, user agent and CDM implementations that allow the CDM to persist data:</p>
-          <ul>
-            <li><p>MUST ensure it is restricted to the <a def-id="origin"></a> for which it was created.</p></li>
-            <li><p>MUST ensure it is restricted to the current <a def-id="browsing-profile"></a> and does not leak to or from private browsing sessions.</p></li>
-            <li><p>MUST allow the user to clear it.</p></li>
-            <li><p>SHOULD allow the user to clear it by <a def-id="origin"></a>.</p></li>
-            <li><p>SHOULD treat it like other site data, including presenting it along with cookies [[!COOKIES]], including it in "remove all data", and presenting it in the same UI locations.</p></li>
-          </ul>
-
           <dl>
-            <dt>Origin-specific Key System storage</dt>
+            <dt>Use Origin-specific and user agent profile-specific Key System storage</dt>
             <dd>
-              <p>Any data persistently stored by the CDM that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific way.
-                Session data, licenses, and keys that are persistently stored MUST be stored per-<a def-id="origin"></a>.
+              <p>Any data persistently stored by the CDM that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific and user agent profile-specific way and MUST NOT leak to or from private browsing sessions.
+                Session data, licenses, and keys that are persistently stored MUST be stored per-<a def-id="origin"></a> and per user agent profile.
                 See <a def-id="session-storage"></a>.
               </p>
             </dd>
-
             <dt>User deletion of Key System storage</dt>
             <dd>
-              <p>User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> and/or for all origins.
+              <p>
+                User Agents MUST allow the user to clear data persisted by the CDM and SHOULD allow users to clear that data by <a def-id="origin"></a>.
+                User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> and/or for all origins.
                 User agents SHOULD help the user avoid <a href="#incomplete-clearing">Incomplete Clearing of Data</a>.
               </p>
             </dd>
 
             <dt>Treating <a def-id="distinctive-identifiers"></a> and Key System stored data like cookies / web storage</dt>
             <dd>
+              <p>
+                User Agents SHOULD treat data persisted by CDMs like other site data, including presenting it along with cookies
```

Comment:
We might want to integrate some of these details into the #like-cookies section referenced below.